app/controllers/idv/in_person/verify_info_controller.rb

```diff
@@ -73,11 +73,23 @@ def flow_session
 
       def analytics_arguments
         {
-          flow_path: flow_path,
+          flow_path: flow_session[:flow_path],
           step: 'verify',
           analytics_id: 'In Person Proofing',
           irs_reproofing: irs_reproofing?,
-        }.merge(**acuant_sdk_ab_test_analytics_args)
+        }.merge(**acuant_sdk_ab_test_analytics_args).
+          merge(**extra_analytics_properties)
+      end
+
+      def extra_analytics_properties
+        extra = {
+          pii_like_keypaths: [[:same_address_as_id], [:state_id, :state_id_jurisdiction]],
+        }
+        unless flow_session[:pii_from_user]&.[](:same_address_as_id).nil?
+          extra[:same_address_as_id] =
+            flow_session[:pii_from_user][:same_address_as_id].to_s == 'true'
```

Comment:
Just curious, but why are we coercing to a string here? Do we not know the type of :same_address_as_id?

Reply:
@JackRyan1989  This bit of code was taken from in_person_flow.rb, I kept it the exact same as is it just missing on analytics when moving verify out of the flow state machine (discovered in the delete PR).  There is a ticket, [LG-9543](https://cm-jira.usa.gov/browse/LG-9543), to refactor all string instances of same_address_as_id to a boolean because there are both. That ticket was started but presented roadblocks. How do you feel about that being handled in ticker LG-9543?

Reply:
Sounds good!

Super nit: Maybe change the "same_address_as_id" to be a string in the test then? This is not a blocking comment.

Reply:
This is not needed.  Analytic data is a boolean. @JackRyan1989 and I jumped on a call.